src/routes/lobby/LobbyView.vue

```diff
@@ -172,6 +201,22 @@ export default {
           console.log(err);
         });
     },
+    subscribeToGame(gameId) {
+      this.loading = true;
+      this.gameStore
+        .requestSubscribe(Number(gameId))
+        .then(() => {
+          this.loading = false;
+        })
+        .catch((err) => {
+          this.loading = false;
+          this.errorLoadingDialogVisible = true;
+          console.log(err);
+        });
+    },
```

Comment:
Don't we already have a subscribe to game function in the game store? Are you able to reuse it?